core/src/test/java/com/graphhopper/routing/AlternativeRouteTest.java

```diff
@@ -44,6 +42,8 @@
 public class AlternativeRouteTest {
     private final FlagEncoder carFE = new CarFlagEncoder();
     private final EncodingManager em = EncodingManager.create(carFE);
+    private final TurnCostExtension turnCostExtension = new TurnCostExtension();
```

Comment:
This test was using edge-based traversal mode but without `TurnWeighting` I had to adjust this otherwise the routes would contain u-turns. This is somewhat nasty unfortunately.

Reply:
Should we write this in the changelog that using edge-based alternative routes is now only possible with enabled turn costs?

Reply:
done